### PR TITLE
hwdata: 0.360 -> 0.363

### PR DESCRIPTION
**Pull request by iblech**
###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [x] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.11 Release Notes (or backporting 22.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2211-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by iblech** [comment]
Note: This change may be affected by https://github.com/NixOS/nix/issues/6572.

`nix-shell -I nixpkgs=... -p hwdata` correctly produces the expected files in `/nix/store/acgi1g8vkllzmlhvi8kla0n0qga853hk-hwdata-0.363`, however it warns `build input /nix/store/5w9wnk36rx4ycyg2p7hf034yviphk9x2-hwdata-0.363 does not exist` and pciutils for instance requires the files to be in the latter location instead of the former.

I don't know how to debug this.

**Comment by LunNova** [comment]
This probably needs to target staging due to the mass rebuild.

File: pkgs/os-specific/linux/hwdata/default.nix [hunk 1]
```diff
@@ -20,7 +20,7 @@ stdenv.mkDerivation rec {
   doCheck = false; # this does build machine-specific checks (e.g. enumerates PCI bus)
 
   outputHashMode = "recursive";
```
Comment (SuperSandro2000) [inline pkgs/os-specific/linux/hwdata/default.nix:22]: This was in https://github.com/iblech/nixpkgs/commit/53df81926a769c262d39d36186e0fea28cc8ec6c turned into a FOD and I can't imagine why we need this here.


```suggestion
```

File: pkgs/os-specific/linux/hwdata/default.nix [hunk 2]
```diff
@@ -20,7 +20,7 @@ stdenv.mkDerivation rec {
...
-  outputHash = "sha256-gkgnHy1XwP87qpQiAm31AIAkxgGm5JYxMBr60kvd+gE=";
+  outputHash = "sha256-AvRjzRometVHJL0HV8FEasTkAFMmHPjT+rcW3mO//Yg=";
```
Comment (SuperSandro2000) [inline pkgs/os-specific/linux/hwdata/default.nix:23]:
```suggestion
```

File: pkgs/os-specific/linux/hwdata/default.nix [hunk 3]
```diff
@@ -19,8 +19,6 @@ stdenv.mkDerivation rec {
 
   doCheck = false; # this does build machine-specific checks (e.g. enumerates PCI bus)
```
Comment (SuperSandro2000) [inline pkgs/os-specific/linux/hwdata/default.nix:21]:
```suggestion
```

**Comment by iblech** [comment]
Awesome, thank you for tending to this pull request, @SuperSandro2000!

I can confirm that building hwdata and pciutils work correctly and without warnings. (I don't quite understand why it didn't work as a FOD, even though it had been that way for quite some time.)

**Pull request merged**